rtv/content.py

```diff
@@ -309,9 +325,7 @@ def __init__(self, name, submissions, loader=default_loader):
         self._submission_data = []
 
     @classmethod
-    def from_name(cls, reddit, name, loader=default_loader):
-
-        display_type = 'normal'
+    def from_name(cls, reddit, name, loader=default_loader, display_type = 'hot'):
 
         if name == 'front':
             return cls('Front Page', reddit.get_front_page(limit=None), loader)
```

Comment:
You can remove this if-case now because it's being handled below. Other than that, the code looks great!